### PR TITLE
Adding warning box re:developer preview only

### PR DESCRIPTION
**Pull request by hflynn**
Adding warning banner to top of 5.1.0-m releases to warn that it is a developer only preview.

### CODE REVIEW

**Comment by sbesson** [comment]
See https://downloads.openmicroscopy.org/omero/5.1.0-m2/index-staging.html and https://downloads.openmicroscopy.org/bio-formats/5.1.0-m2/index-staging.html

**Comment by hflynn** [comment]
Looks as expected to me, pushed fix doesn't seem to change the appearance but maybe it will make Travis happy!

File: bf_downloads.html [hunk 1]
```diff
@@ -54,6 +54,16 @@
             <div class="entry-content" style="margin-left:20px;">
                 <h2>Bio-Formats @VERSION@ Downloads</h2>
                 <p>
+                <div style="background-color: #F5A9A9; margin-left: 20px; margin-right: 20px; padding-bottom: 8px; padding-left: 8px; padding-right: 8px; padding-top: 8px;">
+                <b>WARNING</b><br/>
+                This is a developer-preview release <b>NOT INTENDED FOR
+                PRODUCTION USE</b>. It is provided for the purposes of
```
Comment (mtbc) [inline bf_downloads.html]: "purpose" singular as only one purpose is listed.

**Comment by sbesson** [comment]
:+1: @mtbc @joshmoore are you happy with the latest text. If so, I will replace the 5.1.0-m2 index pages.

**Comment by joshmoore** [comment]
Works for me.

**Comment by sbesson** [comment]
See http://downloads.openmicroscopy.org/omero/5.1.0-m2/ and http://downloads.openmicroscopy.org/bio-formats/5.1.0-m2/. Merging

**Comment by hflynn** [comment]
Can we regenerate the m1 page with this too or not worth it?

**Comment by sbesson** [comment]
Can do

**Comment by hflynn** [comment]
great!

**Comment by hflynn** [comment]
--no-rebase

**Comment by mtbc** [comment]
Text seems fine, thank you.

**Pull request merged**